internal/server/http/handler/v1/tree/handler.go

```diff
@@ -90,10 +107,36 @@ func GetTreeByID(svc service.TreeService) fiber.Handler {
 // @Security		Keycloak
 func GetTreeBySensorID(svc service.TreeService) fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		return getTreeByField(c, "sensor_id", svc.GetBySensorID)
+		ctx := c.Context()
+		id := c.Params("sensor_id")
```

Comment:
Please clone the value of `c.Params`, as I am currently having major problems with this exact error
```suggestion
		id := strings.Clone(c.Params("sensor_id"))
```